src/components/common/navigation/NavigationRoutes.tsx

```diff
@@ -191,6 +191,7 @@ export default function NavigationRoutes() {
                 .then((response) => {
                     if (response.result) {
                         localStorage.isDashboardLoggedIn = true
+                        setActionWithExpiry('dashboardLoginTime', 0)
```

Comment:
if `localStorage.isDashboardLoggedIn` is true then this line won't run. is it expected?